[2022-07-29 18:41:10,710] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataperf.create-baselines manual__2022-07-29T18:41:09.513709+00:00 [queued]>
[2022-07-29 18:41:10,720] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataperf.create-baselines manual__2022-07-29T18:41:09.513709+00:00 [queued]>
[2022-07-29 18:41:10,720] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 18:41:10,721] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-29 18:41:10,721] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 18:41:10,736] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): create-baselines> on 2022-07-29 18:41:09.513709+00:00
[2022-07-29 18:41:10,741] {standard_task_runner.py:52} INFO - Started process 101 to run task
[2022-07-29 18:41:10,744] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataperf', 'create-baselines', 'manual__2022-07-29T18:41:09.513709+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/dataperfr.py', '--cfg-path', '/tmp/tmpwe70h3ta', '--error-file', '/tmp/tmppna_pjo2']
[2022-07-29 18:41:10,744] {standard_task_runner.py:80} INFO - Job 192: Subtask create-baselines
[2022-07-29 18:41:10,807] {task_command.py:371} INFO - Running <TaskInstance: dataperf.create-baselines manual__2022-07-29T18:41:09.513709+00:00 [running]> on host 54cc325577cf
[2022-07-29 18:41:10,881] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataperf
AIRFLOW_CTX_TASK_ID=create-baselines
AIRFLOW_CTX_EXECUTION_DATE=2022-07-29T18:41:09.513709+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-29T18:41:09.513709+00:00
[2022-07-29 18:41:10,883] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-29 18:41:10,884] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python /opt/***/dags/dependency/create_baselines.py']
[2022-07-29 18:41:10,895] {subprocess.py:85} INFO - Output:
[2022-07-29 18:41:11,495] {subprocess.py:92} INFO - Traceback (most recent call last):
[2022-07-29 18:41:11,496] {subprocess.py:92} INFO -   File "/opt/***/dags/dependency/create_baselines.py", line 9, in <module>
[2022-07-29 18:41:11,496] {subprocess.py:92} INFO -     from baselines.datascope_wrapper import ShapleyAppraiser
[2022-07-29 18:41:11,496] {subprocess.py:92} INFO -   File "/opt/***/dags/dependency/baselines/datascope_wrapper.py", line 2, in <module>
[2022-07-29 18:41:11,496] {subprocess.py:92} INFO -     from sklearn import pipeline
[2022-07-29 18:41:11,497] {subprocess.py:92} INFO - ModuleNotFoundError: No module named 'sklearn'
[2022-07-29 18:41:11,524] {subprocess.py:96} INFO - Command exited with return code 1
[2022-07-29 18:41:11,542] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-07-29 18:41:11,545] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=dataperf, task_id=create-baselines, execution_date=20220729T184109, start_date=20220729T184110, end_date=20220729T184111
[2022-07-29 18:41:11,558] {standard_task_runner.py:97} ERROR - Failed to execute job 192 for task create-baselines (Bash command failed. The command returned a non-zero exit code 1.; 101)
[2022-07-29 18:41:11,603] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-07-29 18:41:11,641] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
